staging/src/k8s.io/apiserver/pkg/admission/exclusion/exclusion.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exclusion
+
+import (
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// Included is the set of resources that the expression-based admission controllers
+// should intercept.
+// If a resource is transient, i.e., not persisted in the storage, the resource must be
+// in either Included or Excluded list.
+var Included = sets.New[schema.GroupVersionResource](
```

Comment:
These are all kube-apiserver-specific, so these definitions should be in a kube-apiserver package, and be passed into the generic library / admission library for use. 

Aggregated API servers could do the same to exclude similar virtual APIs they serve from interception

Reply:
Fixed. Thank you.